src/applications/simple-forms/21-0845/pages/authorizerType.js

```diff
@@ -9,11 +9,13 @@ export default {
     authorizerType: {
       'ui:title': (
         <>
-          <h2 className="custom-header authorizer-type vads-u-font-size--h3">
+          <span className="custom-header authorizer-type vads-u-font-family--serif vads-u-font-weight--bold vads-u-font-size--h3">
```

Comment:
from a content structure point of view this feels like it should be an h2. what's the reason for us not using h2 here?

Reply:
Same as above, <h> tags in <legends> get pulled as-is into Review-page and generates a11y headers-hierarchy bugs.